README.md

```diff
@@ -1,7 +1,60 @@
 # Overview
 
-The shardus CLI is a command line tool for managing a Shardeum validator instance used by the Shardeum Validator dashboard. The shardus CLI tool lets you create and manage local test networks running your app. It also collects debug data from a running Shardus app network.
+The Shardus CLI is a command line tool for managing a Shardeum validator instance used by the Shardeum Validator dashboard. The shardus CLI tool lets you create and manage local test networks running your app. It also collects debug data from a running Shardus app network.
```

Comment:
> The shardus CLI tool lets you create and manage local test networks running your app

Yes although the shardus CLI can create local test networks, it's a different tool from the operator-cli. This README refers to the operator-cli, which is used for controlling a single node running on a network and not a network for testing.